concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1779,7 +1779,11 @@ jobs:
                       PREFIX_JOB: "true"
                   EOF
 
-                  cf push
+                  # We only push if not running so stats are not disrupted
+                  # during deployments
+                  if ! cf app graphite-nozzle | grep started; then
```

Comment:
Actually, I think I just figured out a way we may be able to do this. Please hold.